packages/valory/agents/impact_evaluator/tests/test_impact_evaluator.py

```diff
@@ -135,9 +135,21 @@ class BaseTestEnd2EndImpactEvaluatorNormalExecution(BaseTestEnd2EndExecution):
             "value": '{"0": "bafybeiabtdl53v2a3irrgrg7eujzffjallpymli763wvhv6gceurfmcemm", "100": "bafybeid46w6yzbehir7ackcnsyuasdkun5aq7jnckt4sknvmiewpph776q", "50000": "bafybeigbxlwzljbxnlwteupmt6c6k7k2m4bbhunvxxa53dc7niuedilnr4", "100000": "bafybeiawxpq4mqckbau3mjwzd3ic2o7ywlhp6zqo7jnaft26zeqm3xsjjy", "150000": "bafybeie6k53dupf7rf6622rzfxu3dmlv36hytqrmzs5yrilxwcrlhrml2m"}',
         },
         {
-            "dotted_path": f"{__param_args_prefix}.ceramic_db_stream_id",
+            "dotted_path": f"{__param_args_prefix}.default_read_stream_id",
+            "value": "stream_id_e2e",
+        },
+        {
+            "dotted_path": f"{__param_args_prefix}.default_write_stream_id",
             "value": "stream_id_e2e",
         },
+        {
+            "dotted_path": f"{__param_args_prefix}.default_read_target_property",
+            "value": "ceramic_db",
+        },
+        {
+            "dotted_path": f"{__param_args_prefix}.default_write_target_property",
+            "value": "ceramic_db",
+        },
```

Comment:
Are these actually being overridden? Because I think this would not override the defaults from the agent configuration.

Reply:
Yes, you are right, that was my mistake :sweat_smile: